packages/twenty-server/src/database/typeorm/metadata/migrations/1726762935841-addCompostiveColumnToIndexFieldMetadata.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddCompositeColumnToIndexFieldMetadata1726762935841
+  implements MigrationInterface
+{
+  name = 'AddCompositeColumnToIndexFieldMetadata1726762935841';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "metadata"."indexFieldMetadata" ADD "compositeColumn" character varying`,
```

Comment:
Why do we need compositeColumn here? Should be reflected by the fact that indexMetadata has a oneToMany indexFieldMetadata or maybe I misunderstood this column?